lib/msal-node/src/network/HttpClient.ts

```diff
@@ -107,9 +107,8 @@ const networkRequestViaProxy = <T>(
             if (statusCode < 200 || statusCode > 299) {
                 request.destroy();
                 socket.destroy();
-                reject(new Error(`HTTP status code ${statusCode}`));
+                reject(new Error(` Error in connection to proxy ${response.statusCode}, ${response?.statusMessage}`));
```

Comment:
nit: `Error connecting to proxy: ${response.statusCode}, ${response?.statusMessage}`